src/prachi/Assignment_30/FindSecondMaxNumber.java

```diff
@@ -0,0 +1,33 @@
+package prachi.Assignment_30;
+
+/*Assignment - 30 : 4th May'2021
+
+WAP to find Second max from given array
+int[] arr = {10,33,43,55,97,11,3};
+ouput : 55 */
+
+public class FindSecondMaxNumber {
+
+	static int inputArr[]={10,33,43,55,97,11,3};
+
+	static int getSecondMaxNum(int[] arr){  
+		int temp;  
+		for (int i=0; i <= arr.length; i++)   
```

Comment:
This can be handle with one loop


Take looks of below code
Shashank - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/1560/files